packages/config/src/proxy.js

```diff
@@ -54,6 +63,14 @@ function createInsightsProxy({ betaEnv, rootFolder, localChrome, customProxy = [
                     }
                 }
             }] : []),
+            ...proxyRoutes ? Object.entries(proxyRoutes).map(([ route, redirect ]) => ({
+                context: (path) => path.includes(route),
+                target: redirect.host || redirect === 'PORTAL_BACKEND_MARKER' ? target : redirect.host || redirect,
```

Comment:
The last OR does not make sense - you already checking `redirect.host` at the beggining.

Reply:
I forgot to wrap it in parenthesis, it should be `(redirect.host || redirect === 'PORTAL_BACKEND_MARKER') ? target : (redirect.host || redirect)`

Reply:
Then it's the same situation - if redirect.host is true, then it's target.

Reply:
Wrong brackets again! `(redirect.host || redirect) === 'PORTAL_BACKEND_MARKER' ? target : (redirect.host || redirect)` might be better to assign `redirect.host || redirect` to a variable to avoid confusion.